src/sourdough-bread-with-seeds-and-grains.md

```diff
@@ -0,0 +1,47 @@
+# Sourdough bread with seeds and grains
+
+![Sourdough bread with seeds and grains](pix/sourdough-bread-with-seeds-and-grains.webp)
+
+- ⏲️ Prep time: 15 min
+- 🍳 Cook time: 120 min
+
+## Ingredients
+
+**For the preparation:**
+
+- 100g grain of choice (e.g. rye, spelt and wheat)
+- 40g linseed
+- 30g sesame
+- 30g sunflower seeds
+- 600g water
+
+**For the bread:**
+
+- 400g sourdough
+- 200g water (25-35°C)
+- 200g rye flour
+- 300g wheat flour
+- 20g sea salt
+- 2tsp bread spice
+- soaked seeds and grains
+  
+## Directions
+
+1. Preparations:
```

Comment:
Was `Preparations` left here by accident? There isn't a separation so it looks like all the directions are `Preparation`.

If you do keep it and add another heading, I'd suggest similar to above and use:

```
### Preparations
```

Reply:
That was by accident. I will change it to subsections similar to ingredients